webclient/src/forms/RequestPasswordResetForm/RequestPasswordResetForm.tsx

```diff
@@ -0,0 +1,66 @@
+// eslint-disable-next-line
+import React from "react";
+import { connect } from 'react-redux';
+import { Form, Field, reduxForm, change } from 'redux-form'
+
+import Button from '@material-ui/core/Button';
+
+import { InputField, KnownHosts } from 'components';
+import { FormKey } from 'types';
+
+import './RequestPasswordResetForm.css';
+
+const RequestPasswordResetForm = (props) => {
+  const { dispatch, handleSubmit } = props;
+
+  const onHostChange = ({ host, port }) => {
+    dispatch(change(FormKey.RESET_PASSWORD_REQUEST, 'host', host));
+    dispatch(change(FormKey.RESET_PASSWORD_REQUEST, 'port', port));
+  }
+
```

Comment:
Here we need to useReduxEffect for when the resetRequest returns with the demand for an email.